redash/models.py

```diff
@@ -663,13 +670,20 @@ def get_latest(cls, data_source, query, max_age=0):
 
     @classmethod
     def store_result(cls, org, data_source, query_hash, query, data, run_time, retrieved_at):
+        try:
+            data_scanned_information = json.loads(data)['data_scanned']
+        except (ValueError, TypeError) as e:
```

Comment:
You can use `json.loads(data).get('data_scanned', 'error')` which will return `None` if it has no value. No need for the `try`/`except`.

But why make every other query runner return some value here when this is probably relevant for 2-3 data sources at the moment? 

I think a simpler approach will be to return nothing when it's not supported, and just not show any value in the UI.

---

Another issue is that calling `json.loads` here can be a performance/memory issue for large datasets. I have a different approach to suggest, but will outline it in the review summary.